bibtex-actions-file.el

```diff
@@ -124,21 +123,24 @@ If you use 'org-roam' and 'org-roam-bibtex', you can use
             (when file-field
               (seq-mapcat
                (lambda (func)
-                 (funcall func dirs file-field))
+                 (funcall
+                  func
+                  ;; Make sure this arg is non-nil.
+                  (or dirs default-directory)
```

Comment:
I don't think this a good idea, default directory is buffer local. While I think what was needed was to deal with absolute paths. This will add path relative to the default directory which is probably going to be the parent of latex/org file. I guess that can make sense for local bib.

Reply:
In my (non-exhaustive) experiments, if the file path is absolute, it just ignores this, but returns the absolute path.

I still don't like it though; there should be a more clear design that makes clear this will work with absolute paths and nil path variables for the library.

Reply:
> In my (non-exhaustive) experiments, if the file path is absolute, it just ignores the dir arg, but returns the absolute path.
> 

I think empty string is a better choice in that case but it seems like end result will be the same. `expand-file-name` treats the directory argument relative to the `default-directory` if it isn't absolute.